cmd/image-loader/main.go

```diff
@@ -131,6 +132,31 @@ func main() {
 		log.Fatalw("Error loading versions", zap.Error(err))
 	}
 
+	// if no local addons path is given, use the configured addons
+	// Docker image and extract the addons from there
+	if o.addonsPath == "" {
+		v := common.NewDefaultVersions()
+		addonsImage := kubermaticConfig.Spec.UserCluster.Addons.Kubernetes.DockerRepository + ":" + v.Kubermatic
```

Comment:
This panics when no `configuration-file` is given.

Reply:
Good catch!